### PR TITLE
migrate to GH actions

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

**Comment by evgeni** [comment]
I would've expected this would trigger an acrion run, like you can see in https://github.com/evgeni/foreman-ansible-modules/pull/2/ -- weird

**Comment by evgeni** [comment]
this is btw an alternative to #495 :)

**Comment by evgeni** [comment]
(closed the other PR, now it triggered, weird)

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+name: CI
+
+on: [pull_request]
```
Comment (evgeni) [inline .github/workflows/main.yml]: this should also probably include "push to master"

**Comment by evgeni** [comment]
@ehelms I've pushed a few minor changes, mind having a second look before I merge?

**Comment by mdellweg** [comment]
I am kind of pleased to see that the tests are working independently from the chosen CI.

File: tests/extras/sanity.yml [hunk 2]
```diff
@@ -3,7 +3,12 @@
 - hosts: fixtures
   tasks:
     - name: Run ansible-test
-      command: "ansible-test sanity --venv --python {{ lookup('env', 'TRAVIS_PYTHON_VERSION') | default(ansible_python_version.split('.')[:2] | join('.'), true) }} plugins/"
+      command: "ansible-test sanity --venv --python {{ ansible_python['version']['major'] }}.{{ ansible_python['version']['minor'] }} plugins/"
```
Comment (mdellweg) [inline tests/extras/sanity.yml]: Yay!

File: requirements.txt [hunk 3]
```diff
@@ -1,3 +1,2 @@
 apypie>=0.0.5
-ansible>=2.3
```
Comment (mdellweg) [inline requirements.txt]: Isn't that still a requirement?

Comment (evgeni) [reply requirements.txt]: yeah. I needed to remove it from the dev file so I can install different ansible versions from the matrix, but the end user requirement is still valid.

Comment (mdellweg) [reply requirements.txt]: Can you install the other ansible version beforehand? Than this requirement will already be satisfied and not touched again.

File: requirements.txt [hunk 4]
```diff
@@ -1,2 +1,3 @@
 apypie>=0.0.5
```
Comment (mdellweg) [inline requirements.txt]: Btw. Don't we need 0.1.0 here?

Comment (evgeni) [reply requirements.txt]: not strictly, I don't think we *depend* on any 0.1 features (but then again, I didn't test it either)

**Pull request merged**